models/stg_stripe__pricing.sql

```diff
@@ -0,0 +1,120 @@
+{{ config(enabled=var('stripe__using_subscriptions', True)) }}
```

Comment:
Do we need to the conditional blocks in this model? I checked downstream and it looks like the below are the only fields we end up using in our transform models:
- is_active
- unit_amount
- recurring_interval
- recurring_interval_count
- nickname
- product_id

Since all of these fields are available in both `price` and `plan`, we can probably just make this one code block and allow the tmp model to handle the price vs plan piece. What are your thoughts?

If we do go this route, we should update the unique id field to something like `price_id` regardless of plan/price so we don't have to handle the conditional downstream. This is also consistent with the naming of this model.

Reply:
Be sure to update the yml for this table definition with any changes that follow this note.

Reply:
Updated based on our convo, and updated the yml+ docs as well